### DISCUSSION TITLE
Cannot find SEO fields in documentation

### DISCUSSION THREAD

**Discussion by falko100** [payloadcms/payload#673] [Q&A] [answered]
In the demo these fields are available;
![image](https://user-images.githubusercontent.com/11974977/174630264-27622d61-d2e1-4096-a0ba-32b055e906f5.png)

But in the documentation they are nowhere to be found. Is there any documentation available?

**Comment by DanRibbens** [comment, marked as answer]
Good question! The public-demo is a showcase of some of the ways you can customize a Payload project in a unique way. If you want to have anything in your project, just copy the code for it from the public-demo.

The SEO fields specifically are from this plugin: https://github.com/payloadcms/plugin-seo
